README.md

```diff
@@ -2,23 +2,78 @@
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![Node.js CI](https://github.com/pandell/jest-teamcity/actions/workflows/node-ci.yml/badge.svg)](https://github.com/pandell/jest-teamcity/actions/workflows/node-ci.yml)
+[![npm](https://img.shields.io/npm/v/@pandell/jest-teamcity)](https://www.npmjs.com/package/@pandell/jest-teamcity)
 
 `jest-teamcity` is a modern Jest Test Result Processor for TeamCity written in TypeScript. It makes use of TeamCity [service messages](https://www.jetbrains.com/help/teamcity/service-messages.html) to pass information about the Jest test results to the TeamCity server. This makes it possible to display Jest test results in real-time and make the test information available on the `Tests` tab of the `Build Results` page.
 
+## How To Use
+
+To use `@pandell/jest-teamcity`, add the package to your project and update your Jest configuration:
+
+```sh
+yarn add -D @pandell/jest-teamcity
+```
+
+Add a reporter to the Jest configuration file or the `package.json`. For details, check the [Jest documentation](https://jestjs.io/docs/configuration#reporters-arraymodulename--modulename-options).
+
+```js
+// `jest.config.js`
+module.export = {
+    ...
+    reporters: ["@pandell/jest-teamcity"]
```

Comment:
Might be worth mentioning a fallback here so we only use this report on teamcity and not when running directly from the terminal.

```js
reporters: process.env.TEAMCITY_VERSION ? ["@pandell/jest-teamcity"] : ["default"],
```